packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.tsx

```diff
@@ -67,39 +69,43 @@ export function ComboboxContent(props: ComboboxContentProps): JSX.Element {
         />
 
         {searchValue && (
-          <div
-            className={styles.triggerIcon}
-            onClick={() => setSearchValue("")}
+          <button
+            className={styles.clearSearch}
+            onClick={clearSearch}
             data-testid="ATL-Combobox-Content-Search-Clear"
             aria-label="Clear search"
-            role="button"
           >
             <Icon name="remove" size="small" />
-          </div>
+          </button>
         )}
       </div>
+      <div className={styles.list}>
+        {filteredOptions.map(option => (
+          <button
+            className={classnames(
+              styles.option,
+              option.id === selectedOption?.id && styles.selectedOption,
+            )}
+            key={option.id}
+            onClick={() => handleSelection(option)}
+          >
+            {option.label}
+          </button>
+        ))}
 
-      {filteredOptions.map(option => (
-        <button
-          className={classnames(
-            styles.option,
-            option.id === selectedOption?.id && styles.selectedOption,
-          )}
-          key={option.id}
-          onClick={() => handleSelection(option)}
-        >
-          {option.label}
-        </button>
-      ))}
-
-      {filteredOptions.length === 0 && <p>No results for {searchValue}</p>}
-
-      {props.children && <div className={styles.actions}>{props.children}</div>}
+        {filteredOptions.length === 0 && <p>No results for {searchValue}</p>}
```

Comment:
Can we add the " " surrounding the {searchValue} 
![Screenshot 2023-09-08 at 12 05 01 PM](https://github.com/GetJobber/atlantis/assets/104797704/28e82768-325b-49fd-a9ca-ab24d8ec62ed)

Reply:
added this on the other branch!